controllers/org.js

```diff
@@ -58,6 +58,7 @@ exports.newOrg = function(req, res) {
 /**
  * POST /orgs/new
  */
+ //TODO This should move to the survey page after creation. Perplexed a bit.
```

Comment:
@WCRest I tried to have this redirect to the survey page (`/org/id/survey`) next, but it ends up failing with a 404.

If I have this redirect to `/`, I can then navigate to the `/org/id/survey` without issue. I suspect I'm futzing up something javascript-y to do with the callbacks. Any thoughts?

Reply:
I got around this by having "/" redirect a user without a survey to the survey creation page, which I kind of like better (no need to review applications until you have a survey). I'd still love to know what I was getting wrong in this approach though.